.github/workflows/test.yml

```diff
@@ -19,6 +19,10 @@ jobs:
         uses: ./.github/actions/prepare/
       - name: Lint Wiki
         run: yarn lint:check
+      # - name: Build Cli
+      #   run: cd cli && yarn && yarn build && cd ..
+      # - name: Lint IOTA Markdown
+      #   run: yarn lint:md:check
```

Comment:
```suggestion
      # - name: Build Cli
      #   run: yarn ./cli build
      # - name: Lint IOTA Markdown
      #   run: yarn lint:md:check
```

Reply:
We could also consider adding this as a `build:cli` script in the `package.json` and add it to the `prepare` script so it will always be available after the `Prepare Yarn` job.

Reply:
Yeah could make sense. Can you create another PR for that?

Reply:
Willdo